inst/builtin/bs5/shiny/_rules.scss

```
@@ -75,12 +75,18 @@ $bslib-checkbox-radio-margin-right: 0.35em !default;
 
   // Add box-shadow & remove border from 'top-level' card()/layout_sidebar()
   .bslib-card, .bslib-sidebar-layout {
+    // Shadows on top-level cards/sidebars, not in nested components
     box-shadow: $box-shadow;
-    --#{$prefix}card-border-width: 0;
     .bslib-card, .bslib-sidebar-layout {
       box-shadow: none;
+    }
+
+    // Remove outer border, but keep inner borders and borders on nested components
+    --#{$prefix}card-border-width: 0;
+    .card-body, .card-header, .card-footer, .bslib-card, .bslib-sidebar-layout {
```

Any reason why this can't be generic?

```suggestion
    * {
```

---

Or maybe it would make sense to just do `border-width: 0;`?

---

Yeah let's do this second option